gems/nokogiri/CVE-2021-3537.yml

```diff
@@ -0,0 +1,30 @@
+---
+gem: nokogiri
+cve: 2021-3537
+ghsa: 286v-pcf5-25rc
+url: https://bugzilla.redhat.com/show_bug.cgi?id=1956522
```

Comment:
Maybe the primary URL should be `https://github.com/advisories/GHSA-286v-pcf5-25rc` which discusses nokogiri specifically.